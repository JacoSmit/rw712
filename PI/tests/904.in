10
5
7 5 3 13 11
5
7 5 3 11 13
5
11 13 7 3 5
5
11 7 5 13 3
5
5 7 3 11 13
5
11 7 5 13 3
5
13 5 7 3 11
5
7 11 3 13 5
5
3 11 13 7 5
5
13 5 11 3 7
