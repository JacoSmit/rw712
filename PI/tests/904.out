1:
1 ~ 6
2 ~ 2
3 ~ 2
4 ~ 10
5 ~ 6
2:
1 ~ 6
2 ~ 2
3 ~ 2
4 ~ 6
5 ~ 10
3:
1 ~ 6
2 ~ 10
3 ~ 6
4 ~ 2
5 ~ 2
4:
1 ~ 6
2 ~ 6
3 ~ 2
4 ~ 10
5 ~ 2
5:
1 ~ 2
2 ~ 6
3 ~ 2
4 ~ 6
5 ~ 10
6:
1 ~ 6
2 ~ 6
3 ~ 2
4 ~ 10
5 ~ 2
7:
1 ~ 10
2 ~ 2
3 ~ 6
4 ~ 2
5 ~ 6
8:
1 ~ 6
2 ~ 6
3 ~ 2
4 ~ 10
5 ~ 2
9:
1 ~ 2
2 ~ 6
3 ~ 10
4 ~ 6
5 ~ 2
10:
1 ~ 10
2 ~ 2
3 ~ 6
4 ~ 2
5 ~ 6
